app/models/non_hmis_client.rb

```diff
@@ -182,7 +182,7 @@ def set_project_client_fields project_client
     project_client.health_prioritized = health_prioritized
     project_client.hiv_positive = hiv_aids
     project_client.is_currently_youth = current_assessment&.is_currently_youth || false
-
+    project_client.older_than_65 = current_assessment&.older_than_65
```

Comment:
Should this have `|| false` like the line above?

Reply:
no because we don't want to assume people are under 65.  It handles null, whereas the other rule does not.